samples/go/quickstart/quickstart.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2017 Attic Labs, Inc. All rights reserved.
```

Comment:
Can we make this show up directly in the doc somehow? If not, what is the benefit to checking it in?

Reply:
That doesn't appear possible. I wanted to make sure the code compiled so I created a sample and thought it might be nice to have it as a convenience. But it's not that big of a win, I'll delete this PR.